resonance_audio/utils/lockless_task_queue.h

```diff
@@ -17,16 +17,15 @@ limitations under the License.
 #ifndef RESONANCE_AUDIO_UTILS_LOCKLESS_TASK_QUEUE_H_
 #define RESONANCE_AUDIO_UTILS_LOCKLESS_TASK_QUEUE_H_
 
-#include <stddef.h>
-
 #include <atomic>
+#include <cstdint>
 #include <functional>
 #include <vector>
 
 namespace vraudio {
 
-// Lock-less task queue which is thread safe for cocurrent task producers and a
-// single task executor thread.
+// Lock-less task queue which is thread safe for concurrent task producers and
+// concurrent task consumers.
```

Comment:
temp_tasks_ is still being shared by consumers, so this second part isn't currently true.